packages/@ama-sdk/schematics/package.json

```diff
@@ -54,17 +54,24 @@
     },
     "@openapitools/openapi-generator-cli": {
       "optional": true
+    },
+    "openapi-types": {
+      "optional": true
+    },
+    "type-fest": {
+      "optional": true
     }
   },
   "peerDependencies": {
     "@ama-sdk/core": "workspace:^",
     "@angular-devkit/core": "~18.2.0",
-    "@angular-devkit/schematics": "~18.2.0",
     "@angular-devkit/schematics-cli": "^18.0.5",
     "@angular/cli": "~18.2.0",
     "@o3r/schematics": "workspace:^",
     "@openapitools/openapi-generator-cli": "~2.14.0",
-    "@schematics/angular": "~18.2.0"
+    "@schematics/angular": "~18.2.0",
```

Comment:
Not sure this dep is needed for this package

Reply:
We did not indeed